### ISSUE TITLE
"I want to schedule an event" crashes the assistant

### ISSUE THREAD

**Issue by Xyaneon** [WarriorIng64/LTUAssistant#8]
Either speak this or pass this sentence to the assistant, and it crashes with the following traceback:

``` pytb
Traceback (most recent call last):
  File "LTUAssistant.py", line 28, in <module>
    Integrate(" ".join(sys.argv[1:]))
  File "LTUAssistant.py", line 19, in Integrate
    (verb, verb_object, noun2, verb2, preposition) = CoreNLP.Parse(sentence)
  File "/home/christopher/LTUAssistant/CoreNLP.py", line 129, in Parse
    print("Changing verb to %s" % GetWords(sentence, verb))
  File "/home/christopher/LTUAssistant/CoreNLP.py", line 113, in GetWords
    return " ".join(parsed["tokens"][wordPositions[0]:wordPositions[1]])
TypeError: 'int' object has no attribute '__getitem__'
```

**Comment by jacob1** [comment]
Seems like it is an issue with the CoreNLP part of the code, <del>i'll look into it tomorrow.</del>

**Comment by jacob1** [comment]
Fixed in 6368b64d448c4e03c6ca1fa61d9510ac9e1e3c1a

Although if you still want to actually parse the sentence, might possibly need some more changes, or could be doable now.
